data/ooniprobe.conf.sample

```diff
@@ -22,6 +22,7 @@ reports:
     # This is a prefix for each packet capture file (.pcap) per test:
     pcap: null
     collector: null
+    format: yaml
```

Comment:
I had originally added this to support also continued reporting in YAML, but forgot to remove it in the latest version. Good catch.